.github/workflows/post-release.yml

```diff
@@ -10,119 +10,119 @@ env:
-
-      - name: Push branch up
-        if: steps.update-entries.outputs.continue == 'true'
-        run: git push origin prep/post-release-tasks-${{ github.event.release.tag_name }}
-
-      - name: Create the PR
-        if: steps.update-entries.outputs.continue == 'true'
-        uses: actions/github-script@v6
-        with:
-          script: |
-            const body = "This PR updates the changelog.txt entries based on the latest release: ${{ github.event.release.tag_name }}"
-
-            const pr = await github.rest.pulls.create({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              title: "Update changelog.txt from release ${{ github.event.release.tag_name }}",
-              head: "prep/post-release-tasks-${{ github.event.release.tag_name }}",
-              base: "trunk",
-              body: body
-            })
-
-            const prCreated = await github.rest.pulls.requestReviewers({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              pull_number: pr.data.number,
-              reviewers: ["${{ github.event.release.author.login }}"]
-            })
+    changelog-version-update:
+        name: Update changelog and version
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Git fetch trunk branch
+              run: git fetch origin trunk
+
+            - name: Copy readme.txt to vm root
+              run: cp ./plugins/woocommerce/readme.txt ../../readme.txt
+
+            - name: Switch to trunk branch
+              run: git checkout trunk
+
+            - name: Create a new branch based on trunk
+              run: git checkout -b prep/post-release-tasks-${{ github.event.release.tag_name }}
+
+            - name: Check if we need to continue processing
+              uses: actions/github-script@v6
+              id: check
+              with:
+                  script: |
+                      const fs = require( 'node:fs' );
+                      const version = ${{ toJSON( github.event.release.tag_name ) }}
+
+                      fs.readFile( './plugins/woocommerce/readme.txt', 'utf-8', function( err, data ) {
+                        if ( err ) {
+                          console.error( err );
+                        }
+
+                        const regex = /Stable\stag:\s(\d+\.\d+\.\d+)/;
+
+                        const stableVersion = data.match( regex )[1];
+
+                        // If the release version is less than stable version we can bail.
+                        if ( version.localeCompare( stableVersion, undefined, { numeric: true, sensitivity: 'base' } ) == -1 ) {
+                          console.log( 'Release version is less than stable version. No automated action taken. A manual process is required.' );
+                          core.setOutput( 'continue', 'false' )
```

Comment:
Just a heads up, you'll still see the deprecation warning when `core.setOutput` is run because even as of latest it still does a `set-output` command after doing the command to write to `GITHUB_OUTPUT`

Reply:
See https://github.com/actions/toolkit/issues/1218